sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox/unified_metrics_v1/query.py

```diff
@@ -157,7 +157,12 @@ def main():
     update_schema(bq, legacy_core, schema)
     update_schema(bq, legacy_event, schema)
 
-    stripped = [c.split()[0].lstrip("root.") for c in column_summary]
+    # these columns needs to be excluded due to a change in view generation (metrics)
+    # for more details, see: https://github.com/mozilla/bigquery-etl/pull/4029
+    # and https://bugzilla.mozilla.org/show_bug.cgi?id=1741487
+    columns_to_exclude = ("root.metrics.text RECORD", "root.metrics.url RECORD", "root.metrics.jwe RECORD", "root.metrics.labeled_rate RECORD",)
```

Comment:
should this be a constant at the top of the script?

```python
EXCLUDED_COLUMNS = https://github.com/mozilla/bigquery-etl/pull/4029
```

Reply:
I don't think it has to be since it's only used in one place exactly one line below where it's declared. But I also don't have strong objections against making it one if you would prefer.